docs/tutorial/part-seven/index.md

```diff
@@ -198,27 +175,21 @@ exports.createPages = async ({ graphql, actions }) => {
 // highlight-end
 ```
 
-You've added an implementation of the
-[`createPages`](/docs/node-apis/#createPages) API which Gatsby calls so plugins can add
-pages.
+आपने [`createPages`](/docs/node-apis/#createPage) API का कार्यान्वयन ऐड किया है जिसे Gatsby कॉल करता है ताकि प्लगइन्स पेजेज को ऐड कर सकें।
```

Comment:
`पेजेज` => `पेजेज़`

let's fix this everywhere in whole file

Reply:
replaced everywhere